File: assume/app/src/main/java/com/sqisland/espresso/assume/MainActivity.java
package com.sqisland.espresso.assume;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.View;

import java.util.Locale;

public class MainActivity extends AppCompatActivity {
  @Override
  protected void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    setContentView(R.layout.activity_main);

    boolean isInTheUK = Locale.UK.equals(Locale.getDefault());
    View button = findViewById(R.id.button);
    button.setEnabled(isInTheUK);
  }
}

File: assume/app/src/main/java/com/sqisland/espresso/assume/MainActivity.kt
package com.sqisland.espresso.assume

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.view.View
import java.util.Locale

class MainActivity : AppCompatActivity() {
  override fun onCreate(savedInstanceState: Bundle?) {
    super.onCreate(savedInstanceState)
    setContentView(R.layout.activity_main)

    val isInTheUK = Locale.UK == Locale.getDefault()
    val button = findViewById<View>(R.id.button)
    button.isEnabled = isInTheUK
  }
}